src/Aeson.purs

```diff
@@ -379,6 +383,11 @@ instance (GDecodeAeson row list, RL.RowToList row list) => DecodeAeson (Record r
       Just object -> gDecodeAeson object (Proxy :: Proxy list)
       Nothing -> Left $ TypeMismatch "Object"
 
+else instance (InOneOf b a b, DecodeAeson a, DecodeAeson b) => DecodeAeson (a |+| b) where
```

Comment:
[InOneOf](https://pursuit.purescript.org/packages/purescript-untagged-union/0.3.0/docs/Untagged.Union#t:InOneOf) has an instance of form `InOneOf b a b` for any types `a` and `b`, so this seem to be a useless constraint here. Am I right?